lib/miq_automation_engine/service_models/miq_ae_service_service.rb

```diff
@@ -32,6 +31,9 @@ def self.create(options = {})
       ar_method { MiqAeServiceModelBase.wrap_results(Service.create!(attributes)) }
     end
 
+    def retire_service_resources
+    end
```

Comment:
@bdunne Can you just add a comment that this is for backwards compatibility with Automate scripts.  Maybe include a link to https://github.com/ManageIQ/manageiq-content/pull/590.